pallets/handles/src/lib.rs

```diff
@@ -711,8 +713,9 @@ pub mod pallet {
 			base_handle: &str,
 			suffix: HandleSuffix,
 		) -> Result<DisplayHandle, DispatchError> {
+			let base_handle_trimmed = trim_and_collapse_whitespace(base_handle);
```

Comment:
Believe it or not, this appears to be the place that builds the display handle that is saved.